src/pages/student/page.tsx

```diff
@@ -1,11 +1,83 @@
-import { Layout } from "antd";
+import { Link } from "atomic-router-react";
+import { Card, Col, Layout, Row, Space, Typography } from "antd";
+import { useUnit } from "effector-react";
+import { SettingOutlined } from "@ant-design/icons";
 import { PageHeader } from "../../shared/ui/PageHeader/PageHeader";
+import { routes } from "../../shared/lib/atomic-router/route";
+import { $user } from "./model";
+import { homeworkCards } from "./settings/mocks";
+import styles from "./page.module.css";
 
 const { Content } = Layout;
+const { Text, Title } = Typography;
 
-export const StudentPage = () => (
-  <Layout>
-    <PageHeader title="Личный кабинет" />
-    <Content />
-  </Layout>
-);
+export const StudentPage = () => {
+  const [user] = useUnit([$user]);
+
+  return (
+    <Layout>
+      <PageHeader title="Личный кабинет" />
+      <Content className={styles._}>
+        <Row>
+          <Space size={100}>
+            <Col span={24}>
+              <Title className={styles.title}>
+                Привет, {user.name} {user.surname} ({user.nickName})
+              </Title>
+              <Title level={3}>Оценки за дз</Title>
+              <Card className={styles.cardBody}>
+                {homeworkCards.map((card) => (
+                  <Card.Grid key={card.deadline} className={styles.gridStyle}>
+                    <Text
+                    // type={card.homeworkStatus}
+                    >
+                      {card.homework}
+                    </Text>
+                  </Card.Grid>
+                ))}
+              </Card>
+              <Row gutter={20}>
+                {homeworkCards.map((card) => (
+                  <Col key={card.deadline} className={styles.gridStyle}>
+                    <div>{card.homeworkStatus ? "" : card.deadline}</div>
```

Comment:
why div? won't it be better to use card with content in grid system?[https://codesandbox.io/s/7r6080?file=/demo.tsx:249-261](url)